.travis/deb.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash -x
+BUILDPKGS='build-essential autoconf libtool libssl-dev libxt-dev libpq-dev libexpat1-dev libedit-dev libncurses5-dev libical-dev libhwloc-dev pkg-config tcl-dev tk-dev python-dev swig'
+DEPPKGS='postgresql'
+TESTPKGS='python-pip sudo'
+[ "${OS_TYPE}" == "debian:8" ] && ${DOCKER_EXEC} apt-get -qq update || true
+[ "${OS_TYPE}" == "debian:8" ] && ${DOCKER_EXEC} apt-get install -y $BUILDPKGS $DEPPKGS $TESTPKGS || true
+${DOCKER_EXEC} ./autogen.sh
+${DOCKER_EXEC} ./configure
+${DOCKER_EXEC} make
+${DOCKER_EXEC} make install
```

Comment:
We should be creating a .deb package as opposed to running "make install". We may want to use the alien tool to accomplish this.